### PR TITLE
docs: document resource.inclusions setting

### PR DESCRIPTION
**Pull request by alexmt**
Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Optional. My organization is added to the README.
* [x] I've signed the CLA and my build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

File: docs/operator-manual/declarative-setup.md [hunk 1]
```diff
@@ -543,6 +543,22 @@ The `resource.exclusions` node is a list of objects. Each object can have:
 
 If all three match, then the resource is ignored.
 
+In addition to exclusions, you might configure the list of included resources using the `resource.inclusions` setting.
+By default, all resource group/kinds are included. The `resource.inclusions` setting allows customizing the list of included group/kinds:
```
Comment (jannfis) [inline docs/operator-manual/declarative-setup.md]: Is `resource.inclusions` mutually exclusive to the exclusions? Or is it combinable? It's not very clear to me.

**Review by jannfis — CHANGES_REQUESTED** [review]
Just a minor concern, because I don't understand it.

**Comment by alexmt** [comment]
Thank you for review @jannfis! Added paragraph with clarification. Does it look better now?

**Review by jannfis — APPROVED** [review]
Perfect, thank you! Now even I understand it :)

**Pull request merged**